src/openklant2/_resources/betrokkene.py

```diff
@@ -24,7 +28,12 @@ def create(
         response = self._post(self.base_path, data=data)
         return cast(Betrokkene, self.process_response(response))
 
-    def retrieve(self, /, uuid: str | uuid.UUID) -> Betrokkene:
+    def retrieve(
+        self,
+        /,
+        uuid: str | uuid.UUID,
+        params: Optional[BetrokkeneRetrieveParams] = None,
```

Comment:
Let's use new-style annotations:

```suggestion
        params: BetrokkeneRetrieveParams | None = None,
```

Reply:
This one was missed?